cdp_client/cdp.py

```diff
@@ -202,7 +204,7 @@ def report_children_diff():
     def _update_value(self, variant):
         self._value = self._value_from_variant(self._structure.info.value_type, variant)
         for callback in self._value_subscriptions:
-            callback(self._value, variant.timestamp)
+            callback(self._value, variant.timestamp + self._connection.server_time_difference())
```

Comment:
Is `variant.timestamp` in nanoseconds but `self._connection.server_time_difference()` in seconds? Can't find where you do the conversion